tools/targetbrowser/backupplan.go

```diff
@@ -0,0 +1,41 @@
+package targetbrowser
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/google/go-querystring/query"
+	"github.com/thedevsaddam/gojsonq"
+)
+
+// ListOptions for backupPlan
+type ListOptions struct {
+	Page           int    `url:"page"`
+	PageSize       int    `url:"pageSize"`
+	Ordering       string `url:"ordering"`
+	TvkInstanceUID string `url:"tvkInstanceUID"`
+}
+
+// GetBackupPlans returns backupPlan with available options
+func (c *Client) GetBackupPlans(options *ListOptions) error {
+	values, err := query.Values(options)
+	if err != nil {
+		return err
+	}
+	queryParam := values.Encode()
+	req, err := http.NewRequest("GET", fmt.Sprintf("%sbackupplan?%s", c.baseURL, queryParam), nil)
```

Comment:
Declare const for API paths backupplan, backup, etc

Reply:
done